tools/pyinstaller_hooks/hook-rotkehlchen.py

```diff
@@ -1,8 +1,8 @@
 import os
-
 import pkg_resources
 
 datas = []
+processed = []
```

Comment:
as discussed in discord can you make this into a python set?

Also can you add a comment here explaining why this is needed (circular package dependencies)